### PR TITLE
AP_Scripting: Update plane_aerobtics.lua and its README.md

### PR DESCRIPTION
**Pull request by Hwurzburg**
added txt scheds to docs, expanded the info a bit

### CODE REVIEW

File: libraries/AP_Scripting/examples/Aerobatics/Trajectory/README.md [hunk 1]
```diff
@@ -64,7 +68,15 @@ parameters (described below).
 | 203 | FAI F3C Scale Example (left to right)
 | 204 | AirShow
 
-Note: ID's 202-203 are best flown with a mission start point 150m out from the pilot, with the prior and subsequent mission waypoints in a straight line with the model starting teh script flying down wind. ID 201 is best started in teh same manner, but the model positioned 100m out from the pilot.
+Note: ID's 202-203 are best flown with a mission start point 150m out from the pilot, with the prior and subsequent mission waypoints in a straight line with the model starting teh script flying down wind. ID 201 is best started in the same manner, but the model positioned 100m out from the pilot.
```
Comment (tridge) [inline libraries/AP_Scripting/examples/Aerobatics/Trajectory/README.md:71]: typo teh/the

File: libraries/AP_Scripting/examples/Aerobatics/Trajectory/README.md [hunk 2]
```diff
@@ -13,9 +13,11 @@ an appropriate aircraft, and be ready to take manual control if necessary!
 
 The following table gives the available manoeuvres. Each manoeuvre has
 an ID number which is used in the AUTO mission or in the TRIKn_ID
-parameters (described below). The "Turnaround" column indicates if the 
-manoeuvre results in a course reversal, which impacts how it is used in 
-AUTO missions.
+parameters (described below). The present ground track is used as the track for the trick.
+The "Turnaround" column indicates if the manoeuvre results in a course reversal, which impacts how it is used in 
+AUTO missions. Once the trick is completed, the mode that was being used at the start of the trick is restored, including
+the ground track for CRUISE. Tricks in AUTO missions require that they be performed between two waypoints to establish
```
Comment (tridge) [inline libraries/AP_Scripting/examples/Aerobatics/Trajectory/README.md:19]: course and height are unlocked during a trick in CRUISE

File: libraries/AP_Scripting/examples/Aerobatics/Trajectory/plane_aerobatics.lua [hunk 3]
```diff
@@ -2305,7 +2306,7 @@ function load_trick(id)
       -- already have it
       return
    end
-   local fname = string.format("trick%u.txt", id)
+   local fname = string.format("scripts/trick%u.txt", id)
```
Comment (tridge) [inline libraries/AP_Scripting/examples/Aerobatics/Trajectory/plane_aerobatics.lua:2309]: better to try APM/scripts, scripts/ and current directory

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_Scripting/examples/Aerobatics/Trajectory/plane_aerobatics.lua (hunk 3 above)
Comment (Hwurzburg) [reply libraries/AP_Scripting/examples/Aerobatics/Trajectory/plane_aerobatics.lua:2309]: fixed in your PR....removing the lua changes

**Review by tridge — APPROVED** [review]

**Pull request merged**